ecosystem/typescript/sdk_v2/tests/unit/aptos_config.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Aptos, AptosConfig } from "../../src";
+import { Network } from "../../src/utils/api-endpoints";
+
+describe("aptos config", () => {
+  test("it should set TESTNET network if network is not provided", async () => {
```

Comment:
This would be new right, in v1 before you couldn't not specify a network?

I'm not sure this even matters or needs to be addressed now, but if we're going to provide a default network we could consider devnet, depending on how much we care about random data being permanently on testnet

Reply:
Also slightly related: none of this is actually package into the `aptos` package yet, right? So we don't have to worry about breaking anything for the time being until releasing it?

Reply:
> This would be new right, in v1 before you couldn't not specify a network?
> 
> I'm not sure this even matters or needs to be addressed now, but if we're going to provide a default network we could consider devnet, depending on how much we care about random data being permanently on testnet
> 
> But idk, some people use testnet way more than I do so maybe they'd have an opinion about it

Yep, we dont enforce specifying network, and simply set it to a default one. As for what is the default, we can go either way, I remember in the past people pitched for testnet so new devs would not get confused on why their data suddenly disappeared lol but devnet also makes sense, don't have strong opinion here.

Reply:
another new thing is that we initialize with a Network enum which is a "name" and not with a network url as in v1

Reply:
Ah okay, makes sense to just keep it the same then! I didn't realize it already had a default value.

And yes, I love it!it works with the indexer too now :D